src/index.tsx

```diff
@@ -1,4 +1,4 @@
 import ReactDOM from 'react-dom';
 import App from './App';
-
-ReactDOM.render(<App />, document.getElementById('root'));
+// eslint-disable-next-line
+ReactDOM.render(<App todos={[]} user={null} />, document.getElementById('root'));
```

Comment:
```suggestion
ReactDOM.render(<App />, document.getElementById('root'));
```
Зачем передавать сюда эти пропсы?